core/base/inc/TTimer.h

```diff
@@ -80,6 +80,7 @@ class TTimer : public TSysEvtHandler {
    Bool_t         IsSync() const { return fSync; }
    Bool_t         IsAsync() const { return !fSync; }
    Bool_t         IsInterruptingSyscalls() const { return fIntSyscalls; }
+   Bool_t         IsRunning();///< Check if timer is running within gSystem (has been started and did not finish yet).
```

Comment:
To match the rest of the file, the documentation string for the function should be in the source file.

Reply:
Done